src/pages/Login/index.tsx

```diff
@@ -35,49 +35,52 @@ const Login = ({ history }: any) => {
 
   const submitHandler = (e: React.FormEvent) => {
     e.preventDefault()
-    console.log('login')
-    // dispatch login(email, password)
+    if (jobSeeker) {
+      dispatch(loginJobSeekerRequest(email, password))
+    } else if (employer) {
+      dispatch(loginEmployerRequest(email, password))
+    }
   }
 
   return (
     <div>
       <NavBar />
       <FormContainer>
-        <h2 className='signin'>Sign In To TinDev</h2>
-        {/* {error && <Message variant="danger">{error}</Message>} */}
+        <h2 className="signin">Sign In To TinDev</h2>
+        {error && <Message variant="danger">{error}</Message>}
         {loading && <Loader />}
         <Form onSubmit={submitHandler}>
-          <Form.Group controlId='email'>
-            <Form.Label className='label'>EMAIL</Form.Label>
+          <Form.Group controlId="email">
+            <Form.Label className="label">EMAIL</Form.Label>
             <Form.Control
-              type='email'
-              placeholder='Enter email'
+              type="email"
+              placeholder="Enter email"
               value={email}
-              onChange={(e) => setEmail(e.target.value)}
+              onChange={e => setEmail(e.target.value)}
             ></Form.Control>
           </Form.Group>
-          <Form.Group controlId='password'>
-            <Form.Label className='label'>PASSWORD</Form.Label>
+          <Form.Group controlId="password">
+            <Form.Label className="label">PASSWORD</Form.Label>
             <Form.Control
-              type='password'
-              placeholder='Enter password'
+              type="password"
+              placeholder="Enter password"
               value={password}
-              onChange={(e) => setPassword(e.target.value)}
+              onChange={e => setPassword(e.target.value)}
             ></Form.Control>
           </Form.Group>
-          <CustomButton text={login} />
+          <CustomButton text="Login" />
         </Form>
-        <Row className='forgot-password py-3'>
+        <Row className="forgot-password py-3">
           <Col>
-            <Link className='forgot-password' to='/reset-password'>
+            <Link className="forgot-password" to="/reset-password">
               Forgot your password?
             </Link>
           </Col>
         </Row>
-        <Row className='new-user'>
+        <Row className="new-user">
           <Col>
             New User?{' '}
-            <Link className='register' to='/register'>
+            <Link className="register" to="/register">
               Register
             </Link>
           </Col>
```

Comment:
I don't see any button to choose to login as `employer` or `jobSeeker`, where are you controlling those two state ?

Reply:
In the navbar.

Reply:
When you're finished, can you update the pr with picutes of new pages as well ?